AttitudeManager/ControlAlgorithms/Src/AM_FixedControl.cpp

```diff
@@ -10,13 +10,13 @@ std::vector<ActuatorOutput> FixedControl::runControlsAlgorithm(
     const AttitudeManagerInput &instructions) {
  
    // Current attitude from SF
-    LosSFData current_attitude {}; 
+    LOS::LosSFData current_attitude {}; 
     
     // Compute target values
     float target_heading = instructions.heading;
-    float target_throttle = instructions.speed; 
-    float target_pitch = instructions.x_dir * MAX_PITCH_ANGLE;
-    float target_bank = instructions.y_dir * MAX_BANK_ANGLE;  
+    float target_throttle = instructions.velocity_target; 
```

Comment:
keep in mind that velocity seems to denote a vector. Speed is magnitude only, which is why we chose it in the first place.

I'd discuss with @Gfcello and @Aidan-B.

Reply:
tagging @kozihkode because she suggested the change to velocity_target from speed.

Reply:
decision to revert to speed was made